app/views/notifiers/course/forum/post_notifier/replied/user_notifications/email.html.slim

```diff
@@ -7,7 +7,7 @@
 
 - course_title = format_inline_text(course.title)
 - topic_title = format_inline_text(topic.title)
-- post_author = format_inline_text(post.creator.name)
+- post_author = format_inline_text(course.course_users.for_user(post.creator).first.name)
```

Comment:
Let's shift `course_user = course.course_users.for_user(post.creator).first` into a variable above. 

We also need to guard against a missing `course_user` (ie. admins), so probably if that variable doesn't exist, we fallback to the user `post.creator.name`. Could be assigned to a variable like `post_author_name` to make it a little more readable also.